notebook/static/custom/custom.js

```diff
@@ -72,3 +72,20 @@
  * @class customjs
  * @static
  */
+
+define(function() {
```

Comment:
Can we please add a comment on why this function would be required ?

Reply:
Added comment explaining its function.